pdr_backend/lake/table_bronze_pdr_predictions.py

```diff
@@ -46,8 +46,7 @@ def get_bronze_pdr_predictions_data_with_SQL(
         f"""
         SELECT
             {pdr_predictions_table_name}.ID as ID,
-            SPLIT_PART({pdr_predictions_table_name}.ID, '-', 1)
-                || '-' || SPLIT_PART({pdr_predictions_table_name}.ID, '-', 2) AS slot_id,
+            {pdr_predictions_table_name}.pdr_slot_slot_id as slot_id,
```

Comment:
This line is trowing me an error. We try to access "pdr_slot_slot_id" from `pdr_predictions` but that field it's not set